src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -71,6 +71,46 @@ export class ServiceFormComponent extends CdForm implements OnInit {
   action: string;
   resource: string;
   serviceTypes: string[] = [];
+  // Options with user friendly labels
+  serviceTypeOptions: Array<{ value: string; label: string }> = [];
+  // Grouped options
+  serviceTypeOptionGroups: Array<{ label: string; options: Array<{ value: string; label: string }> }> = [];
+
+  // Map of known service type -> user friendly label. Fallback will be generated
+  readonly SERVICE_TYPE_LABELS: { [key: string]: string } = {
+    rgw: 'Object(RGW)',
+    nfs: 'NFS',
+    iscsi: 'iSCSI',
+    nvmeof: 'NVMe/TCP',
+    'mgmt-gateway': 'Management Gateway',
+    'oauth2-proxy': 'OAuth2 Proxy',
+    smb: 'SMB',
+    ingress: 'Ingress Controller',
+    grafana: 'Grafana',
+    mds: 'Filesystem(MDS)',
+    mon: 'Ceph Monitor',
+    mgr: 'Ceph Manager',
+    'rbd-mirror': 'Block Mirroring',
+    'cephfs-mirror': 'Filesystem Mirroring',
+    loki: 'Grafana Loki',
+    alloy: 'Grafana Alloy',
+    container: 'Container',
+    osd: 'OSD',
+  };
```

Comment:
I think this is a relatviely simple change that we can make it cephadm easily.

Reply:
Correct, part is who does it and when? I assume everyone is aware about bad naming, but we haven't taken action on it.  Do you think our team should take that responsibility?